_commands/basics/uniq.md

```diff
@@ -36,6 +36,26 @@ _Notice that the 2 that follows the 3 is not filtered, even though it is repeate
  * This is the default action if no flag is specified
  * The `-u` option tells `uniq` to filter out only those lines that are unique, with non-consecutive repeated lines being treated as unique instances
  * Refer to the example above. No flag is specified, so the deafult `-u` option is used
+ * The `-u` option is most commonly used with [sort](/commands/sort) to first sort a file and then remove consecutive duplicated lines (see example below)
+
+~~~ bash
+$ cat numbers.txt
+2
+2
+0
+2
+1
+1
+3
+4
+4
+$ numbers.txt | sort | uniq -u
```

Comment:
Isn't `uniq -u` just `uniq`? If so, don't make it more confusing than it has to be.